examples/demo/modules/blog/cms_blog_api.e

```diff
@@ -8,93 +8,108 @@ class
 	CMS_BLOG_API
 
 inherit
-	CMS_NODE_API
```

Comment:
in this case, I think it is safer to split the different API and have proper blog api, and node api.